src/OrleansRuntime/ConsistentRing/VirtualBucketsRingProvider.cs

```diff
@@ -5,6 +5,8 @@
 
 namespace Orleans.Runtime.ConsistentRing
 {
+    using Orleans.Runtime.Configuration;
```

Comment:
Good point. I just let ReSharper/StyleCop do what it wants most of the time and only revert their autoformatting if it's a minor change. I can revert bits if people have a strong opinion.